### PR TITLE
KAKFA-17173: move quota config params from KafkaConfig to QuotaConfig

### PR DESCRIPTION
**Pull request by unknowntpo**
This PR renamed `QuotaConfigs` to `QuotaConfig`, and
moved quota config related variable from `KafkaConfig` to `QuotaConfig`.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by unknowntpo** [comment]
@chia7712 Would you like to review this PR ?

File: tools/src/main/java/org/apache/kafka/tools/reassign/ReassignPartitionsCommand.java [hunk 1]
```diff
@@ -1409,8 +1409,8 @@ static ReassignPartitionsCommandOptions validateAndParseArgs(String[] args) {
         }
 
         OptionSpec<?> action = allActions.get(0);
-        
-        if (opts.options.has(opts.bootstrapServerOpt) && opts.options.has(opts.bootstrapControllerOpt)) 
+
+        if (opts.options.has(opts.bootstrapServerOpt) && opts.options.has(opts.bootstrapControllerOpt))
```
Comment (m1a2st) [inline tools/src/main/java/org/apache/kafka/tools/reassign/ReassignPartitionsCommand.java]: Please revert this  unnessary change

**Review by m1a2st — COMMENTED** [review]
Hello @unknowntpo, Thanks for this PR, left a small comment.

File: tools/src/main/java/org/apache/kafka/tools/reassign/ReassignPartitionsCommand.java (hunk 1 above)
Comment (unknowntpo) [reply tools/src/main/java/org/apache/kafka/tools/reassign/ReassignPartitionsCommand.java]: @m1a2st Done.

**Review by chia7712 — APPROVED** [review]

**Pull request merged**